Automattic/wp-calypso
# Accept invite: Validate email when creating passwordless account

Related to #87379
Fixes #87379

## Proposed Changes

* Implement email validation on passwordless invitation form. It should validate already existing users and any type of wrong email.
If the email already exists, it should show a message explaining that the email already exists, and offer links to log in or reset the password. After logging in, the user should be back to accept invite form.


## Testing Instructions



- Use live link or apply this branch locally
- From your atomic site, invite an user
- Get the invite in the email and accept the invitation
- When landing on WPCom type a wrong/malformed email 
- It should validate if the email is correct and disable the continue button
- If you use an existing email, it should present you options to login or reset password
- If you log in, it should bring you back to the accept invitation form again to accept or not the invitation.


## Pre-merge Checklist
